### PR TITLE
Refactoring

### PR DESCRIPTION
**Pull request by pinglinh**
Have tried to refactor code however only test 1, 2 and 3 pass. Not sure where am I going wrong with the code.

### CODE REVIEW

**Comment by pinglinh** [comment]
- 14 tests are now passing
- IV, IX, X, XIV, XIX, XX are still not passing

File: lib/roman_numerals.rb [hunk 1]
```diff
@@ -1,19 +1,21 @@
...
+    	romanValue << "X"
+    	remaining_number -= 10
     end
+
+	if remaining_number >= 5
+	  romanValue << "V"
+	  remaining_number -= 5
+	end
+	   
+    romanValue << "I" * remaining_number
```
Comment (cnak) [inline lib/roman_numerals.rb:17]: Perhaps at this point, you could've exposed the duplication?
Make it similar to the two above.

``` ruby
if remaining_number >= 1
      romanValue << "I"
      remaining_number -= 1
end
```

File: spec/roman_numerals_spec.rb [hunk 2]
```diff
@@ -39,4 +44,64 @@
...
+
+    it 'returns XIX for 19' do
+	  result = arabic_to_roman.convert(19)
+	  expect(result).to eq "XIX"
+    end
+
+    it 'returns XX for 20' do
+	  result = arabic_to_roman.convert(20)
+	  expect(result).to eq "XX"
+    end
```
Comment (cnak) [inline spec/roman_numerals_spec.rb]: I don't think these tests will pass.
Please don't commit to the repo if any tests are failing.

**Review by cnak — CHANGES_REQUESTED** [review]
Please either get the tests to pass or remove the tests

**Pull request merged**